lib/gateway/Shard.js

```diff
@@ -1250,12 +1090,7 @@ class Shard extends EventEmitter {
                         this.client.token = "Bot " + this.client.token;
                     }
                 } else {
-                    this.client.bot = false;
-                    this.client.userGuildSettings = {};
-                    packet.d.user_guild_settings.forEach((guildSettings) => {
-                        this.client.userGuildSettings[guildSettings.guild_id] = guildSettings;
-                    });
-                    this.client.userSettings = packet.d.user_settings;
+                    throw "User bots are not allowed by Discord's Terms of Service, please use a bot account.";
```

Comment:
Throwing strings is bad practice, they don't contain stack traces. `throw new Error()` pls.
```suggestion
                    throw new Error("User bots are not allowed by Discord's Terms of Service, please use a bot account.");
```